Sources/Sentry/SentryClient.m

```diff
@@ -168,11 +169,17 @@ - (SentryId *)captureError:(NSError *)error
 - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithLevel:kSentryLevelError];
-    NSString *errorCodeAsString = [NSString stringWithFormat:@"%ld", (long)error.code];
-    SentryException *sentryException = [[SentryException alloc] initWithValue:errorCodeAsString
+    NSString *exceptionValue = [NSString stringWithFormat:@"Code: %ld LocalizedDescription: %@",
+                                         (long)error.code, error.localizedDescription];
+    SentryException *exception = [[SentryException alloc] initWithValue:exceptionValue
                                                                          type:error.domain];
-    event.exceptions = @[ sentryException ];
-    event.fingerprint = @[ error.domain, [NSString stringWithFormat:@"%ld", (long)error.code] ];
+
+    // Sentry uses the error on the mechanism for gouping
+    SentryMechanism *mechanism = [[SentryMechanism alloc] initWithType:@"NSError"];
+    mechanism.error = error;
```

Comment:
Oh I was wonder what this was :)

https://develop.sentry.dev/sdk/event-payloads/exception/#ns_error

Reply:
This is brand new 🔥😁